package.json

```diff
@@ -1,23 +1,4 @@
 {
-  "name": "performance",
-  "version": "1.0.0-beta.2",
-  "private": true,
-  "description": "Performance plugin from the WordPress Performance Group, which is a collection of standalone performance modules.",
-  "author": "The WordPress Contributors",
-  "license": "GPL-2.0-or-later",
```

Comment:
When running `npm install`, there's a warning on the `license` field missing. There's no harm in keeping that as it will never change, so I'd be in favor of keeping it.

Reply:
Ok, added it back although it is not needed and we can ignore those warnings.